docs/reference/setup/sysconfig/virtual-memory.asciidoc

```diff
@@ -18,3 +18,10 @@ To set this value permanently, update the `vm.max_map_count` setting in
 
 The RPM and Debian packages will configure this setting automatically. No
 further configuration is required.
+
+You can find out the current mmap count of a running Elasticsearch process using the following command:
+[source,sh]
+----------------------------
+ls /proc/$PID/maps | wc -l
```

Comment:
Sorry I gave you duff info here, `/proc/$PID/maps` is just a file:

```suggestion
wc -l /proc/$PID/maps
```

Reply:
I would have tried it first. But that doesn't work on Mac!